.cursor/skills/dev-helper/scripts/send-pr.sh

```
@@ -0,0 +1,125 @@
+  IFS=',' read -ra LABEL_ARRAY <<< "$PR_LABELS"
+  for label in "${LABEL_ARRAY[@]}"; do
+    LABEL_FLAGS+=" --label $label"
+  done
+fi
+
+pr_url=$(gh pr create \
+  --repo "$GH_REPO" \
+  --head "$current_branch" \
+  --title "$TITLE" \
+  --body-file "$BODY_FILE" \
+  $DRAFT_FLAG $LABEL_FLAGS < /dev/null 2>&1)
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)